src/module/displayTodo.js

```diff
@@ -53,6 +54,19 @@ const displayTodo = (todo) => {
   deleteIcon.textContent = 'delete';
   li.appendChild(deleteIcon);
 
+  checkbox.addEventListener('click', () => {
+    p.style.textDecoration = 'line-through';
+    complete(todo.index);
+    checkbox.classList.add('hide');
+    check.classList.remove('hide');
+  });
```

Comment:
- [ ] Good job on your implementation so far but after marking a todo item as completed, I am not able toggle it back to false. Please ensure that when the checkbox is clicked it can be toggled between true and false. 

Hint:
- You can use the `change` event listener instead of `click`. Please find the docs [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)